openmc/capi/core.py

```diff
@@ -73,8 +73,21 @@ class _Bank(Structure):
 _dll.openmc_statepoint_write.argtypes = [c_char_p, POINTER(c_bool)]
 _dll.openmc_statepoint_write.restype = c_int
 _dll.openmc_statepoint_write.errcheck = _error_handler
+_dll.openmc_global_bounding_box.argtypes = [POINTER(c_double),
+                                            POINTER(c_double)]
+_dll.openmc_global_bounding_box.restype = c_int
+_dll.openmc_global_bounding_box.errcheck = _error_handler
 
 
+def global_bounding_box():
+    """Calculate a global bounding box for the model"""
+    llc = np.zeros(3)
+    urc = np.zeros(3)
+    _dll.openmc_global_bounding_box(llc.ctypes.data_as(POINTER(c_double)),
+                                    urc.ctypes.data_as(POINTER(c_double)))
+
```

Comment:
Question about these capi calls @paulromano. 

I use `sys.float_info.max` to check if these values are infinite in the tests. Should I instead look for those values here and replace them with `np.inf` before returning the bounding box?

Something like:

```Python
inf = sys.float_info.max
llc[llc == inf] = np.inf
urc[urc == inf] = np.inf
llc[llc == -inf] = -np.inf
urc[urc == -inf] = -np.inf
```

There may be a cleaner way of doing that...

Reply:
Ah right, I forgot that the value we are using for infinity is not IEEE-754 infinity. Yes, your suggested code there looks like the way to go.